tasks/apply-mapping/tests/test-apply-mapping-staged-files-expansion.yaml

```diff
@@ -6,8 +6,8 @@ metadata:
 spec:
   description: |
     Run the apply-mapping task with a snapshot.spec json and a custom mapping provided in
-    the data file with tags per component and verify that the resulting json
-    contains the expected values with tags expanded.
+    the data file with staged files per component and verify that the resulting json
```

Comment:
why this change to the description?

Reply:
Because it seemed obvious this description was copied from another test and the author forgot to change it - there are no tags per component in this test. This test is testing the staged files expansion.